Adds note that KSPM doesn't support govcloud (#4942)
elastic/security-docs: docs/cloud-native-security/kspm-get-started.asciidoc

--
* The KSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
* The KSPM integration only works in the `Default` Kibana space. Installing the KSPM integration on a different Kibana space will not work.

> * KSPM is supported on commercial cloud only. Government cloud is not supported (https://github.com/elastic/enhancements[request support]).

Author: ```suggestion
* KSPM is not supported on EKS clusters in AWS Government Cloud (https://github.com/elastic/enhancements[request support for government cloud]).
```
